utils/k8s.go

```diff
@@ -0,0 +1,178 @@
+package utils
+
+import (
+	"context"
+	"fmt"
+	"github.com/digitalocean/godo"
+	log "github.com/sirupsen/logrus"
+	v12 "k8s.io/api/core/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/watch"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	"os"
+	"strings"
+	"time"
+)
+
+type FullNode struct {
```

Comment:
FullNode means?...NodeInfos? If yes, may be renaming it to make it more clear is better

Reply:
done